ui/storybook/src/utils/requests/queryRangeRequests.ts

```diff
@@ -0,0 +1,44 @@
+import { rest, RequestHandler } from 'msw';
+import qs from 'qs';
+
+type MockQueryRangeQueryConfig = {
+  query: string;
+  response: {
+    status?: 200;
+    body: Record<string, unknown>;
+  };
+};
+
+type MockQueryRangeConfig = {
+  queries: MockQueryRangeQueryConfig[];
+};
+
+/**
+ * Mock responses from '/query_range' by the query parameter in the
+ * request. Useful for stabilizing charts when taking screenshots.
+ */
+export function mockQueryRangeRequests({ queries }: MockQueryRangeConfig): RequestHandler[] {
```

Comment:
This is very similar to a util we have in the e2e tests but written to work with MSW (what we're using in storybook to mock api requests) instead of the playwright routing api.